main.tf

```diff
@@ -98,6 +98,11 @@ resource "google_healthcare_fhir_store" "fhir_stores" {
         schema_config {
           schema_type               = lookup(stream_configs.value.bigquery_destination.schema_config, "schema_type", null)
           recursive_structure_depth = stream_configs.value.bigquery_destination.schema_config.recursive_structure_depth
+
+          last_updated_partition_config {
```

Comment:
@FaithfulZhening to confirm, you've tested creating a FHIR store with this field set via Terraform, right?